README.md

```diff
@@ -74,3 +74,26 @@ Use the manager to parse and save a gzipped man page in raw format:
     python runserver.py
      * Running on http://127.0.0.1:5000/
      * Restarting with reloader
+
+### Start up a local web server with docker
+
+    Build docker web and db containers
+    $ docker-compose build
+    $ docker-compose up
+
+    Copy dump over to container for than to import it.
+    $ docker cp dump/ explainshell_db_1:/tmp/dump
+
+    Import classifiers
+    $ docker exec explainshell_db_1 mongorestore /tmp/dump
+
+    Open browser at port 5000
+    $ open http://localhost:5000
```

Comment:
Yes, I think so. I think it can be configured other platforms.

Maybe remove it?